ui-participant/src/hub/kit/KitsPage.tsx

```diff
@@ -0,0 +1,155 @@
+  const currentStep = getStepFromStatus(kit.status)
+  const currentStatusIndex = statusOrder.indexOf(currentStep)
+  const stepIndex = statusOrder.indexOf(step)
+  return stepIndex <= currentStatusIndex
+}
+
+const MailedKitStatusBar = ({ kit }: { kit: KitRequest }) => {
+  return (
+    <div className="progress-stacked border-top border-bottom border-start" style={{ height: '50px' }}>
+      <ProgressBar
+        width="33%" complete={getStepCompletion(kit, 'PREPARING')} icon={faBoxesPacking} label={'Preparing'}/>
+      <ProgressBar
+        width="34%" complete={getStepCompletion(kit, 'SHIPPED')} icon={faTruckFast} label={'Shipped'}/>
+      <ProgressBar
+        width="33%" complete={getStepCompletion(kit, 'RETURNED')} icon={faCircleCheck} label={'Returned'}/>
+    </div>
+  )
+}
+
+const InPersonKitStatusBar = ({ kit }: { kit: KitRequest }) => {
+  return (
+    <div className="progress-stacked" style={{ height: '50px' }}>
+      <ProgressBar
+        width="50%" complete={getStepCompletion(kit, 'PREPARING')} icon={faBoxesPacking} label={'Created'}/>
+      <ProgressBar
+        width="50%" complete={getStepCompletion(kit, 'COLLECTED')} icon={faCircleCheck} label={'Collected'}/>
+    </div>
+  )
+}
+
+const ProgressBar = ({ icon, label, width, complete }: {
+  icon: IconDefinition, label: string, width: string, complete: boolean
+}) => {
+  return (
+    <div className="progress" role="progressbar" style={{ width, height: '50px' }}>
+      <div className={`progress-bar border-end ${complete ? '' : 'bg-dark-subtle'}`}
+        data-testid={`${label.toLowerCase()}-${complete}`}
+        style={{ background: 'var(--brand-color)' }}>
+        <div className="d-flex flex-column align-items-center justify-content-center h-100">
+          <FontAwesomeIcon icon={icon} className={'fa-xl'}/>
+          {label}
+        </div>
+      </div>
+    </div>
+  )
+}
+
+const EnrolleeKitRequests = ({ enrollee }: { enrollee: Enrollee }) => {
+  const enrolleeKitRequests = enrollee.kitRequests
+  const visibleKitRequests = enrolleeKitRequests.filter(kit =>
+    kit.status !== 'DEACTIVATED' && kit.status !== 'ERRORED').sort((a, b) => b.createdAt - a.createdAt)
+
+  //TODO make this a studyenv setting
+  const isInPersonKitEnabled = true
+
+  return <div className="mb-3 rounded round-3 py-4 bg-white px-md-5 shadow-sm px-2">
+    <h1 className="pb-3">Sample collection kits</h1>
+    <div className="pb-4">
+      Sample collection kits are a valuable part of the study process and can help provide researchers with
+      important insights. Below you will find the status of all kits that have been provided to you.
+    </div>
+    { isInPersonKitEnabled &&
+      <>
+        <h3>Provide a sample in-person</h3>
+        <div className="pb-2">
+          This study is currently offering the option to complete a sample collection kit in-person.
+          Click below for more information about this process.
```

Comment:
Is the `Click below` supposed to direct them to click `Complete a kit in-person`? I was a bit confused because I thought I should be looking for a link to a page with documentation. Maybe `Click below if you are attending an in-person event with <study name>`?

Reply:
That was the intention, but I think I'll get rid of the entire "Click below for more information about this process.", it seems redundant